STTApplication/src/main/java/com/build/labs/controller/STTController.java

```diff
@@ -21,118 +21,142 @@
 
 @Controller
 public class STTController {
-	
+
 	private final STTService sttService;
-	
-	
+
 	public STTController(STTService sttService) {
 		super();
 		this.sttService = sttService;
 	}
 
 	/*
-	 * This method build the home page. This page can be accessible in different ways
-	 * {"/", "", "/index", "/home"}, it returns the content in page index.html resides
-	 * in src/main/resources/templates directory.
+	 * This method build the home page. This page can be accessible in different
+	 * ways {"/", "", "/index", "/home"}, it returns the content in page index.html
+	 * resides in src/main/resources/templates directory.
 	 */
-	@GetMapping({"/", "", "/index", "/home"})
-    public String homepage() {
-        return "index";
-    }
+	@GetMapping({ "/", "", "/index", "/home" })
+	public String homepage() {
+		return "index";
+	}
 
 	/*
-	 * This method handles the file upload and convert it to a transcript and return the transcript
-	 * to index page with model attribute result and if result holds any value
-	 * index page displays the result.
+	 * This method handles the file upload and convert it to a transcript and return
+	 * the transcript to index page with model attribute result and if result holds
+	 * any value index page displays the result.
 	 */
-	@PostMapping(value= "/uploadFile", produces = {MediaType.APPLICATION_JSON_VALUE})
+	@PostMapping(value = "/uploadFile", produces = { MediaType.APPLICATION_JSON_VALUE })
 	public ResponseEntity<List<Output>> uploadFile(@RequestParam("file") MultipartFile multipartFile)
 			throws IOException, URISyntaxException {
 
 		try (InputStream inputStream = multipartFile.getInputStream()) {
 			String transcript = sttService.transcriptAudio(inputStream);
 			List<Output> outputList = formatOutput(transcript);
 			return ResponseEntity.ok(outputList);
-			
+
 		} catch (IOException ioe) {
 			throw new IOException("Could not upload file: ", ioe);
 		}
 	}
-	
-	
+
 	@GetMapping("/transcript/{filename}")
-	public String transcriptAudio(@PathVariable("filename") String filename, Model model) throws IOException, URISyntaxException {
-		
-		InputStream input = new ClassPathResource("static/audio/"+filename).getInputStream();
-		String transcript = sttService.transcriptAudio(input);
-		
-		List<Output> outputList = formatOutput(transcript);
-		outputList.forEach(o -> {
-			System.out.println("confidence: " + o.getConfidence());
-			System.out.println("transcript: " + o.getTranscript());
-		});
-		
-		model.addAttribute("result", outputList);
-		return "index";
+	public String transcriptAudio(@PathVariable("filename") String filename, Model model)
+			throws IOException, URISyntaxException {
+
+		InputStream input = new ClassPathResource("static/audio/" + filename).getInputStream();
+		InputStream input1 = new ClassPathResource("static/audio/" + filename).getInputStream();
+		String transcript = "";
+		String transcriptParam = "";
+		if (filename.startsWith("samples")) {
```

Comment:
Can you create your a separate method instead hardcoded filenames? Otherwise in future if anybody wants to change there will be a lot of work.